src/pages/blog-post-template-page/BlogPostTemplatePage.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import * as Bs from 'react-bootstrap';
```

Comment:
We have a unnecessary global import here, only import the relevant bootstrap components like on line